lib/index.js

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) 2018-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import './config.js';
+
+export {default as CredentialBase} from '../components/CredentialBase.vue';
+export {default as CredentialField} from '../components/CredentialField.vue';
+export {default as DynamicImage} from
+  '../components/DynamicImage.vue';
+export {useCredentialCommon} from '../components/credentialCommon.js';
+
+export {default as CredentialSwitch} from '../components/CredentialSwitch.vue';
+
+export const registerComponent = async ({app, component, name = ''}) => {
+  app.component(`${name}`, component);
```

Comment:
Are we expecting `name` to not be a string? If not:

```suggestion
  app.component(name, component);
```

I would think `name` should only ever be a string (or it would be an error).

Reply:
9801dd68bcde6a1ad29042a986b1752c596b4562